doc/examples/color_exposure/plot_local_equalize.py

```diff
@@ -168,3 +168,4 @@ def plot_img_and_hist(image, axes, bins=256):
 
 # prevent overlap of y-axis labels
 fig.tight_layout()
+plt.show()
```

Comment:
@grlee77 I just recalled that @rfezzani suggested removing this line of code in the previous code review, so I removed it from both 2D and 3D demos in this file - I am curious about the difference of having it versus not? 

On a side note, this equalization rank filter demo exists in both color_exposure/plot_local_equalize.py and applications/plot_rank_filters.py; is this (and potentially other rank filter demos) repetition created on purpose? @jni  (we only added the equalization demo in this file)

Reply:
I think the request was just to remove the extra `plt.show()` from the middle of the file (between the 2D and 3D demo). I think there should still be a final `plt.show()` at the end of the file. This makes it so that the figures don't just pop up and then immediately disappear if the example is run from the command line.

Reply:
Thank you!